--- bmon-2.2.0-pre1.orig/configure.ac	2009-08-11 12:39:11.172098193 +0200
+++ bmon-2.2.0-pre1/configure.ac	2009-08-11 13:13:15.358098088 +0200
@@ -192,19 +192,6 @@
 ##
 #####################################################################
 NL="No "
-AC_CHECK_LIB(nl, nl_connect, [
-	AC_DEFINE_UNQUOTED(HAVE_NL, "1", [have libnl])
-	LIBNL="-lnl"
-	NL="Yes"
-],[
-	case ${target_os} in
-	    *linux*)
-		echo
-		echo "*** Warning: Building bmon on Linuxx w/o libnl ***"
-		echo
-	    ;;
-	esac
-])
 
 #####################################################################
 ##
