--- bmon-2.2.0-pre1.orig/src/attr.c	2009-08-11 12:39:11.174098517 +0200
+++ bmon-2.2.0-pre1/src/attr.c	2009-08-11 12:39:37.249973382 +0200
@@ -33,7 +33,7 @@
 struct attr_map attrs[] = {
 	[BYTES] = { .am_name = "bytes" },
 	[PACKETS] = { .am_name = "packets" },
-	[ERRORS] = { .am_name "errors" },
+	[ERRORS] = { .am_name = "errors" },
 	[DROP] = { .am_name = "drop" },
 	[FIFO] = { .am_name = "fifo" },
 	[FRAME] = { .am_name = "frame" },
--- bmon-2.2.0-pre1.orig/src/out_xml_event.c	2009-08-11 12:39:11.174098517 +0200
+++ bmon-2.2.0-pre1/src/out_xml_event.c	2009-08-11 12:52:26.458973199 +0200
@@ -135,7 +135,7 @@
 	.om_draw = xml_event_draw,
 	.om_set_opts = xml_event_set_opts,
 	.om_probe = xml_event_probe,
-	.om_shutdown xml_event_shutdown,
+	.om_shutdown = xml_event_shutdown
 };
 
 static void __init xml_event_init(void)
--- bmon-2.2.0-pre1.orig/src/out_audio.c	2009-08-11 12:39:11.174098517 +0200
+++ bmon-2.2.0-pre1/src/out_audio.c	2009-08-11 12:52:53.630973273 +0200
@@ -141,7 +141,7 @@
 	.om_draw = audio_draw,
 	.om_set_opts = audio_set_opts,
 	.om_probe = audio_probe,
-	.om_shutdown audio_shutdown,
+	.om_shutdown = audio_shutdown
 };
 
 static void __init audio_init(void)
